packages/core/workers/src/cpuCount.js

```diff
@@ -1,11 +1,71 @@
-const os = require('os');
+// @flow
+import os from 'os';
+import {execSync} from 'child_process';
 
-module.exports = function() {
-  let cores;
+const exec = (command: string): string => {
   try {
-    cores = require('physical-cpu-count');
-  } catch (err) {
-    cores = os.cpus().length;
+    let stdout = execSync(command, {
+      encoding: 'utf8',
+      // This prevents the command from outputting to the console
+      stdio: [null, null, null],
```

Comment:
This silences the output, so we don't get any errors or warnings in the console from this util